dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -7782,15 +7782,15 @@ public String toString() {
           .setDescription("The maximum time for a netty client (for block "
               + "reads and block writes) to wait for a response from the data server.")
           .build();
-
+  // TODO(jiacheng): Actually, by being the main switch for dora, this deserves a better name
```

Comment:
Actually, there might be some other places we want to switch on Dora by default. Please just push to this branch @JiamingMai @dbw9580 . Let's try to include the main switching changes in this PR so we either have Dora or no Dora.